### ISSUE TITLE
Add warning info for PiTR experimental feature

### ISSUE THREAD

**Issue by db-will** [pingcap/docs#6565]
File: [/br/use-br-command-line-tool.md](https://docs.pingcap.com/tidb/stable/use-br-command-line-tool)

For the current doc, we have suggested a way to run PiTR with br & TiCDC. However, this feature is still experimental, not fully tested, and still in development. we need to add warning message in the doc to ensure everyone is aware of that.

Use the following msg, as we usually added for other experimental features
```
This feature is experimental and not thoroughly tested. It is highly not recommended to use this feature in the production environment.
```

**Comment by dveeden** [comment]
This seems related to https://github.com/pingcap/docs/pull/6190

That PR says this is a prototype and *not* (yet) experimental

**Comment by yahonda** [comment]
I think this issue can be closed because #7500 has been merged to master, which removes PITR features from BR.
